pixl_export/README.md

```diff
@@ -37,8 +37,36 @@ pytest
 
 ## Usage
 
-Usage should be from the CLI driver, which calls the HTTP endpoints.
+Usage should be from the [CLI driver](../cli/README.md), which calls the HTTP endpoints.
```

Comment:
I think this assumes a lot of understanding of the CLI and PIXL in general.

Again, if there is a desire for the different modules in PIXL to be used independently, then it would be good for the pixl_export documentation to explain how to use it.

For now, I think it would be good to put a few lines in this Usage section which demonstrate it woulds on the command line.

Reply:
noted